pkg/chunk/local/boltdb_index_client.go

```diff
@@ -8,6 +8,10 @@ import (
 	"os"
 	"path"
 	"sync"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/util"
+	"github.com/go-kit/kit/log/level"
```

Comment:
We tend to split imports into 3 sections: stdlib, non-cortex, and cortex.  Could you put these two imports in the right place please?